app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DishesController < ApplicationController
+  def show
+    @dish = Dish.find(params[:id])
+  end
+  
+  def update
```

Comment:
The content of this controller action is correct, but generally we want to reserve this `dishes#update` action for when we are updating an attribute of a dish. In this case, we are creating a joins record (or a relationship) between an existing dish and an existing ingredient. 

So, a more RESTful design would be to create a new joins controller called the DishIngredientsController and you can put this content in a `create` action there. Since you are *creating* a record in the joins table! Let me know if this makes sense or if you want to chat more about this RESTful design choice